.npmrc

```diff
@@ -1,2 +1,2 @@
 registry=http://registry.npmjs.org/
-@zowe:registry=https://api.bintray.com/npm/ca/brightside/
+@zowe:registry=https://gizaartifactory.jfrog.io/gizaartifactory/api/npm/npm-release/
```

Comment:
Does this have to be giza now that its on public npm? Won't this fail for most users?

Also, I've seen in the APIML repo that they include an access key here... this way folks can at least download from giza.

Reply:
I agree with pointing to public NPM (i.e. removing the zowe scope declaration all-together)

And I was under the impression that that was sort of illegal/wrong to open giza to public access, hence why we never included one.

I will remove the zowe scope form there and re-run tests.
Thanks.

Reply:
That is great thanks. I'm not sure if its cool to do this or not, but here is the sample:
https://github.com/zowe/sample-spring-boot-api-service/blob/master/gradle.properties